Resources/Prototypes/_Sunrise/Entities/Objects/Misc/identification_cards.yml

```
@@ -46,9 +46,10 @@
   name: security pilot ID card
   components:
   - type: Sprite
+    sprite: _Sunrise/Objects/Misc/id_cards.rsi
     layers:
     - state: default
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 5

(indentation)